omics/analyzer/omics-run-analyzer

```diff
@@ -0,0 +1,460 @@
+#!/usr/bin/env python3
+"""
+Generate statistics for a completed HealthOmics workflow run
+
+Usage: omics-run-analyzer [<runId>...]
+                          [--profile=<profile>]
+                          [--region=<region>]
+                          [--time=<interval>]
+                          [--show]
+                          [--timeline]
+                          [--file=<path>]
+                          [--out=<path>]
+                          [--help]
+
+Options:
+ -p, --profile=<profile>  AWS profile
+ -r, --region=<region>    AWS region
+ -t, --time=<interval>    Select runs over a time interval [default: 1day]
```

Comment:
Can help text for the `-t` option provide additional acceptable units? I needed to look through the code to know that this is:
```
m|min|minutes?|h|hours?|d|days?|w|weeks
```

Reply:
Sure.

Reply:
This is still missing, but is minor and can be resolved in a follow-up